src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -70,7 +70,7 @@ def handle_exception(e):
 
         @app.before_request
         def log_request_info():
-            app.logger.info("Headers: %s", request.headers)
-            app.logger.info("Body: %s", request.get_data())
+            app.logger.debug("Headers: %s", request.headers)
```

Comment:
let's change the pfs start --syncronize to --debug, so we have a way to print debug log?